mteb/tasks/Classification/zho/CMTEBClassification.py

```diff
@@ -220,21 +220,7 @@ class JDReview(AbsTaskClassification):
         annotations_creators=None,
         dialect=None,
         text_creation=None,
-        bibtex_citation="""@inproceedings{nielsen-2023-scandeval,
-    title = "{S}cand{E}val: A Benchmark for {S}candinavian Natural Language Processing",
-    author = "Nielsen, Dan",
-    editor = {Alum{\"a}e, Tanel  and
-      Fishel, Mark},
-    booktitle = "Proceedings of the 24th Nordic Conference on Computational Linguistics (NoDaLiDa)",
-    month = may,
-    year = "2023",
-    address = "T{\'o}rshavn, Faroe Islands",
-    publisher = "University of Tartu Library",
-    url = "https://aclanthology.org/2023.nodalida-1.20",
-    pages = "185--201",
-    abstract = "This paper introduces a Scandinavian benchmarking platform, ScandEval, which can benchmark any pretrained model on four different tasks in the Scandinavian languages. The datasets used in two of the tasks, linguistic acceptability and question answering, are new. We develop and release a Python package and command-line interface, scandeval, which can benchmark any model that has been uploaded to the Hugging Face Hub, with reproducible results. Using this package, we benchmark more than 80 Scandinavian or multilingual models and present the results of these in an interactive online leaderboard, as well as provide an analysis of the results. The analysis shows that there is substantial cross-lingual transfer among the the Mainland Scandinavian languages (Danish, Swedish and Norwegian), with limited cross-lingual transfer between the group of Mainland Scandinavian languages and the group of Insular Scandinavian languages (Icelandic and Faroese). The benchmarking results also show that the investment in language technology in Norway and Sweden has led to language models that outperform massively multilingual models such as XLM-RoBERTa and mDeBERTaV3. We release the source code for both the package and leaderboard.",
-}
-""",
+        bibtex_citation="",  # missing
```

Comment:
Maybe it should cite the paper that created CMTEB (https://arxiv.org/abs/2309.07597) 🤔